massa-execution-exports/src/test_exports/config.rs

```diff
@@ -43,6 +44,19 @@ impl Default for ExecutionConfig {
             max_datastore_value_size: MAX_DATASTORE_VALUE_LENGTH,
             storage_costs_constants,
             max_read_only_gas: 100_000_000,
+            gas_costs: GasCosts::new(
+                concat!(
+                    env!("CARGO_MANIFEST_DIR"),
+                    "/../massa-node/base_config/gas_costs/abi_gas_costs.json"
```

Comment:
I'm wondering if yaml files would be nicer here as we can add comments in them?

Reply:
I don't think we need to have comments on this as nobody will edit it + it's pretty straightforward. I prefer stay on JSON for that is more readable.